api/src/pcapi/core/offerers/api.py

```diff
@@ -2489,6 +2489,19 @@ def invite_member(offerer: models.Offerer, email: str, current_user: users_model
     transactional_mails.send_offerer_attachment_invitation([email], offerer, existing_user)
 
 
+def invite_member_again(offerer: models.Offerer, email: str) -> None:
+    existing_invited_email = (
+        models.OffererInvitation.query.filter(models.OffererInvitation.offererId == offerer.id)
```

Comment:
Peux-tu désormais utiliser `db.session.query` ? Ici et dans les autres requêtes de la PR.
```suggestion
        db.session.query(models.OffererInvitation)
        .filter(models.OffererInvitation.offererId == offerer.id)
```